vote/src/vote_account.rs

```diff
@@ -891,4 +992,153 @@ mod tests {
             }
         }
     }
+
+    #[test]
+    fn test_clone_and_filter_for_alpenglow_truncates() {
+        let mut rng = rand::thread_rng();
+        let current_limit = 3000;
+        let (vote_accounts, identity_balances) =
+            new_staked_vote_accounts(&mut rng, current_limit, current_limit, None);
+        // All vote accounts should be returned if the limit is high enough.
+        let filtered = vote_accounts.clone_and_filter_for_alpenglow(
+            current_limit + 500,
+            1,
```

Comment:
might be nice to have some `const MIN_IDENTITY_ACCOUNT_BALANCE: usize = 1;` that can be re-used for these tests

Reply:
done.